{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2018-14645", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "A flaw was discovered in the HPACK decoder of HAProxy, before 1.8.14, that is used for HTTP/2. An out-of-bounds read access in hpack_valid_idx() resulted in a remote crash and denial of service.", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 14:43:27 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2019-07-23", "cve_detail": "", "references": [{"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14645", "Description": "CONFIRM:https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14645"}, {"URL": "https://www.mail-archive.com/haproxy@formilux.org/msg31253.html", "Description": "MLIST:[haproxy] 20180920 [ANNOUNCE] haproxy-1.8.14 - Security Update"}, {"URL": "https://access.redhat.com/errata/RHBA-2019:0028", "Description": "REDHAT:RHBA-2019:0028"}, {"URL": "https://access.redhat.com/errata/RHSA-2018:2882", "Description": "REDHAT:RHSA-2018:2882"}, {"URL": "https://usn.ubuntu.com/3780-1/", "Description": "UBUNTU:USN-3780-1"}], "cve_repaire": "", "editor": "warcup", "add_time": "2018-09-21"}